package.json

```diff
@@ -192,7 +192,7 @@
     "decode-uri-component": "^0.2.2",
     "express": "^4.18.2",
     "lodash": "~4.17.21",
-    "minimatch": "~3.1.2",
+    "minimatch": "~5.1.6",
```

Comment:
is this a big jump?

Reply:
Yeah kind of but we only use this package as a dependency for various other packages, mainly test and lint related packages. Since all our tests are green and UI is working it should be safe